src/pages/reader.vue

```diff
@@ -38,13 +38,8 @@ export default {
       return this.$route.query.classId;
     },
     fileSrc() {
-      const { src } = this.$route.query;
-      // TODO: check src exists in ISCN
-      // if (src && this.iscnContentUrls.find(url => url === src)) {
-      if (src) {
-        return src;
-      }
-      return this.iscnContentUrls.find(url => url.includes('pdf'));
+      const { format: type } = this.$route.query;
+      return this.iscnContentUrls.find(url => url.includes(type));
```

Comment:
It is possible that one iscn contains more than 1 pdf/epub, maybe use index as well?

Reply:
[🐛 Fix invalid file-src](https://github.com/likecoin/liker-land/pull/1525/commits/66850b5abd068b9227c6ac4b9e29c39181b0e427)

Reply:
I mean this won't work if there is like 2 content urls
```
- ar://epub_ep1
- ar://epub_ep2
```
If we only check for `epub`, people can never access the 2nd url @AuroraHuang22

Reply:
[🎨 Add 'index' to query for retrieving specific 'fileSrc'](https://github.com/likecoin/liker-land/pull/1525/commits/b1ed3419a3610947acdb39a388219979b5f0c326)